model/qti/ParserFactory.php

```
@@ -1421,6 +1423,12 @@ protected function buildTemplatedrivenResponse(DOMElement $data, $interactions)
         $patternFeedbackMatchChoiceWithElse  = '/responseCondition [count(./*) = 2 ]' . $subPatternFeedbackMatchChoice
             . $subPatternFeedbackElse;
 
+        $patternFeedbackIsOperatorScore = '//responseCondition/responseIf/and[
```

PHPMD (2.15.0)

Avoid excessively long variable names like $patternFeedbackIsOperatorScore. Keep variable name length under 20. (Naming Rules)

(LongVariable)